docs/java-rest/high-level/document/delete.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[java-rest-high-document-delete]]
+=== Delete API
+
+[[java-rest-high-document-delete-request]]
+==== Delete Request
+
+The most simple Delete Request needs is:
+
+["source","java",subs="attributes,callouts"]
+--------------------------------------------------
+sys2::[perl -ne 'exit if /end::delete-request/; print if $tag; $tag = $tag || /tag::delete-request/' {docdir}/../../client/rest-high-level/src/test/java/org/elasticsearch/client/DocumentationIT.java]
```

Comment:
Let's keep it simple for now but that would be lovely to have a gradle task which does that extraction for instance and generate all the doc so we don't call anymore `build_docs.pl` but `gradle docs` or something.